packages/@guardian/source-react-components/src/button/styles.ts

```diff
@@ -260,17 +278,19 @@ export const buttonStyles =
 		iconSide = 'left',
 		nudgeIcon,
 		cssOverrides,
+		isLoading,
 	}: SharedButtonProps) =>
 	(
 		theme: Theme,
 	): Array<string | SerializedStyles | SerializedStyles[] | undefined> =>
 		[
 			button,
 			sizes[size],
-			priorities[priority](theme.button),
-			iconSvg ? iconSizes[size] : '',
-			iconSvg && !hideLabel ? iconSides[iconSide] : '',
+			priorities[priority](theme.button, isLoading),
```

Comment:
Do we still need to pass the `isLoading` param?

```suggestion
			priorities[priority](theme.button),
```